File: close10/close10.rb
def close10(a,b)
    if (((a - 10).abs) == ((b - 10).abs))
        return 0
    else
        if ((a - 10).abs > ((b - 10).abs))
            return b
          end
    return a
    end
end

puts(close10(2,16))

File: close10/close10.php
<?php
function close10($a,$b){
    if ((abs($a - 10)) == (abs($b - 10))){
        echo "0";
    }
    else {
        if (abs($a - 10) > (abs($b - 10))){
            echo $b;
        }else{
            echo $a;
        }
    }
}

close10(2,18);
